src/components/profile/settings/contact-info.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+    <div class="m-auto w-100">
+        <div class="d-flex">
+            <h5 class="m-0 flex-grow-1 align-self-center">contact</h5>
+            <button class="btn align-self-center m-0 p-0 me-2" @click="showModal">
+                <PhPencil size="30" color="#ff5e45" />
+            </button>
+        </div>
+        <div class="pe-2">
+            <div v-for="(info, index) in formattedContactInfo" :key="index">
+                <div class="d-flex gap-1 mb-1 align-items-start">
+                    <div class="d-flex flex-column">
+                        <h6>{{ info.type }}</h6>
+                        <div class="d-flex gap-1">
+                            <div>{{ valueToDisplay(index) }}</div>
+                            <h5>{{ info.isPrimary }}</h5>
+                        </div>
+                    </div>
+                    <div class="d-flex ms-auto my-auto me-0">
+                        <ContactVerificationModal :contactForm="info" :index="index" v-if="!info.isVerified">
+                            <div class="my-auto mx-0 align-self-center ms-1 me-2">
+                                <PhWarning :size="24" color="#f31b65" weight="fill"/>
+                            </div>
+                        </ContactVerificationModal>
+                        <button class="btn m-0 p-0 lh-1" @click="toggleMaskedStatus(index)">
+                            <PhEye :size="20" color="#ff5e45" v-show="info.isMasked"/>
+                            <PhEyeSlash :size="20" color="#ff5e45" v-show="!info.isMasked"/>
+                        </button>
```

Comment:
Make sure flyarrow `warning` color is added to `_fly_arrow_variables.scss`
```scss
$warning: #f31b65;
$warning-rgb: 243, 27, 101;
```

Use bootstrap buttons:
```xml
                        <ContactVerificationModal :contactForm="info" :index="index" v-if="!info.isVerified">
                            <button class="btn btn-icon text-warning fs-4 ms-1 me-2">
                                <PhWarning weight="fill" />
                            </button>
                        </ContactVerificationModal>
                        <button class="btn btn-icon text-primary fs-5" @click="toggleMaskedStatus(index)">
                            <PhEye v-show="info.isMasked" />
                            <PhEyeSlash v-show="!info.isMasked" />
                        </button>
```

Reply:
Worked on this.